### PR TITLE
adds some more lobby music

### PR DESCRIPTION
**Pull request by UselessTheremin**
# im too lazy to fix anything rn so im doing this instead, sleeper featurecoder theremin

### Intent of your Pull Request

adds some more music to the lobby, i like variety and these are game ost's that i think work well on the lobby screen

### Why is this change good for the game?

its not, i have shite music choice

:cl:  
rscadd: more new lobby songs
/:cl:

### CODE REVIEW

**Comment by athena413** [comment]
ok but what are the songs

**Comment by UselessTheremin** [comment]
> ok but what are the songs

check the files changed it has the names and the links

**Comment by JamieD1** [comment]
@UselessTheremin Atomize next time. 

1,2,4 are fine.

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -45,6 +45,10 @@
 		"https://www.youtube.com/watch?v=iZB8XXYePy0",						//The Kinks - Super Sonic Ship
 		"https://www.youtube.com/watch?v=tRcPA7Fzebw",						//David Bowie - Starman
 		"https://www.youtube.com/watch?v=FH2EgYq_NCY",						//Lou Reed - Satellite of Love
+		"https://www.youtube.com/watch?v=2M-XEHh7bu0",						//Death Road To Canada - Rigor Mortis Rag
+		"https://www.youtube.com/watch?v=aYeLVR2xiyc",						//Ridiculon - Walls of Denial
+		"https://www.youtube.com/watch?v=gRinrFL_BbU",						//Throne II Theme
+		"https://www.youtube.com/watch?v=WOplPqnG2CU",						//Noita - Beat of the Cosmos
```
Comment (JamieD1) [inline yogstation/code/controllers/subsystem/ticker.dm:51]:
```suggestion
		"https://www.youtube.com/watch?v=2M-XEHh7bu0",						//Death Road To Canada - Rigor Mortis Rag
		"https://www.youtube.com/watch?v=aYeLVR2xiyc",						//Ridiculon - Walls of Denial
		"https://www.youtube.com/watch?v=WOplPqnG2CU",						//Noita - Beat of the Cosmos
```

Comment (JamieD1) [inline yogstation/code/controllers/subsystem/ticker.dm]:
```suggestion
		"https://www.youtube.com/watch?v=2M-XEHh7bu0",						//Death Road To Canada - Rigor Mortis Rag
		"https://www.youtube.com/watch?v=aYeLVR2xiyc",						//Ridiculon - Walls of Denial
		"https://www.youtube.com/watch?v=WOplPqnG2CU",						//Noita - Beat of the Cosmos
```

Comment (JamieD1) [reply yogstation/code/controllers/subsystem/ticker.dm]: This looks a bit weird, but remove nuclear throne.

**Pull request closed without merging**